lib/charms/nginx_ingress_integrator/v0/ingress.py

```diff
@@ -82,9 +82,7 @@
 OPTIONAL_CACHE_SETTING_RELATION_FIELDS = {
     "refresh-patterns",
 }
-JSON_RELATION_FIELDS = {
-    "cache-settings"
-}
+JSON_RELATION_FIELDS = {"cache-settings"}
```

Comment:
You shouldn't be changing this library, or at least you shouldn't be changing it here but in https://github.com/canonical/nginx-ingress-integrator-operator